test/e2e/nsx_networkinfo_test.go

```diff
@@ -262,16 +262,16 @@ func assureNamespaceDeleted(t *testing.T, ns string) {
 	return
 }
 
-func getVPCPathFromNetworkInfo(t *testing.T, ns, networkInfoName string) (networkInfo *v1alpha1.NetworkInfo) {
+func getVPCSnatIPFromNetworkInfo(t *testing.T, ns, networkInfoName string) (networkInfo *v1alpha1.NetworkInfo) {
```

Comment:
Maybe rename this function as `getNetworkInfoWithPrivateIPs` because the return type is `*v1alpha1.NetworkInfo` but not VPC SNAT IP ?